### PR TITLE
Task1.3 solution

### PR DESCRIPTION
**Pull request by Vomchick**

### CODE REVIEW

File: Task1.3/Program.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Input number of elemets");
+            var arr = new int[int.Parse(Console.ReadLine())];
+            Console.WriteLine($"Input {arr.Length} elements");
+            for (int i = 0; i < arr.Length; i++)
+            {
+                arr[i] = int.Parse(Console.ReadLine());
+            }
+
```
Comment (IgorKheidorov) [inline Task1.3/Program.cs:42]: Code style

File: Task1.3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            foreach (int i in arr) Console.Write(i + " ");
+            Console.WriteLine();
+
+            var set = SetCreation(arr);
```
Comment (IgorKheidorov) [inline Task1.3/Program.cs:50]: Code style, body of the loop in brackets

File: Task1.3/Program.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
+﻿namespace Task1._3
+{
+    internal class Program
+    {
+        static int[] SetCreation(int[] arr)
+        {
+            int count = 0;
```
Comment (IgorKheidorov) [inline Task1.3/Program.cs:7]: No need to iterate via array twice, first loop is just to calculate the size of final array (set). Better to copy unique values to a temp array with the same size as original, calculate actual length in the same loop as copy operation, and than use the truncated array as resulting

File: Task1.3/Program.cs [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            for (int i = 0; i < arr.Length; i++)
+            {
+                for (int j = 0; j < i; j++)
+                {
+                    if (arr[j] == arr[i])
+                    {
+                        repeat = true;
+                        break;
+                    }
+                }
```
Comment (IgorKheidorov) [inline Task1.3/Program.cs:19]: code style

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
See comments

File: Task1.3/Program.cs (hunk 1 above)
Comment (Vomchick) [reply Task1.3/Program.cs:48]: What do you mean by a code style in this case?

**Review by IgorKheidorov — APPROVED** [review]

**Pull request merged**